### DISCUSSION TITLE
Unable to configure roboshop-ansible. (Error 403)

### DISCUSSION THREAD

**Discussion by PranathiAT** [learndevopsonline/learndevopsonline#835] [Q&A] [answered]
Hi @r-devops ,

   I am unable to configure roboshop-ansible .Please check the screenshots below.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129154029/7c774326-388e-4d01-8fb4-102139781c66)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/129154029/89441d95-26c3-4bea-8895-be53514186e8)

**Comment by anandaiahk** [comment]
This solution is safe to use
We came along this issue when we changed Jenkins to be accessible via a reverse proxy.

There is an option in the "Configure Global Security" that "Enable proxy compatibility"

This helped with my issue.

Enter image description here

**Comment by anandaiahk** [comment, marked as answer]
goto csrf protection and goto the crumb algorithm and click enable proxy compatibility

**Reply by PranathiAT** [reply]
Thank you. Its working.
